packages/terra-menu/src/MenuItem.jsx

```diff
@@ -349,7 +355,7 @@ class MenuItem extends React.Component {
     } else if (toggleable) {
       role = 'menuitemcheckbox';
     }
-
+    const menuRole = MenuUtils.isMac() ? role : 'menuitem';
```

Comment:
instead of declaring new variable try adding check for above condition blocks 
const isMacOs = MenuUtils.isMac();
if (isGroupItem && isMacOs) {
      role = 'menuitemradio';
    } else if (toggleable && isMacOs) {
      role = 'menuitemcheckbox';
    }

Reply:
Updated role check - [339656d](https://github.com/cerner/terra-framework/pull/1718/commits/339656d46490613a1c21e937904e476e954f6a7a)